.github/scripts-windows/run-tests-python2.cmd

```diff
@@ -1,3 +1,3 @@
 cd runtime-testsuite
-mvn -Dantlr-python2-exec="C:\Python27\python.exe" -Dtest=python2.** test
+mvn -Dantlr-python2-exec="%PYTHON_HOME%\python.exe" -Dtest=python2.** test
```

Comment:
We don't need this to be portable, currently it only runs on 4 docker instances on a physical box in my office

Reply:
A hardcoded path won't wok on Github cloud though. Python is not installed on that path.

Just may be - We might be on the different page here. Please follow comments on #3783 if you haven't yet.

Reply:
Is it possible to use just different environment variables `%PYTHON2_HOME%` and `%PYTHON3_HOME%` to make it workable everywhere?

Reply:
Sure. @ericvergnaud Would that resolve your concern?

Reply:
It might, but we don't need to solve problems we don't have. The existing scripts work just fine, they're just not portable, which is not something we need at this point.
Sorry for the tone, it's great that you've gotten the cpp build to run on Windows. I suggest you restrict this PR to just that.